sympy/matrices/tests/test_commonmatrix.py

```diff
@@ -1469,7 +1468,6 @@ def test_singular_values():
     assert A.T.singular_values() == \
         [sqrt(sqrt(221) + 15), sqrt(15 - sqrt(221)), 0, 0]
 
-
```

Comment:
And here

Reply:
i don't know why my quality check said failed,so I was trying to figure out and finally when I did,I forgot to revert back changes.